packages/website/ts/pages/governance/ledger_sign_note.tsx

```diff
@@ -0,0 +1,50 @@
+import * as _ from 'lodash';
+import * as React from 'react';
+import styled from 'styled-components';
+
+import { colors } from 'ts/style/colors';
+
+interface LedgerSignNoteProps {
+    text?: string;
```

Comment:
do we want this to be optional?

Reply:
I did it like that because of also passing them to the styled-component, which only uses `isVisible` - but is there a better pattern?

Is this better?
```
interface WrapperProps {
	isVisible: boolean;
}
interface LedgerSignNoteProps extends WrapperProps {
	text: string;
}
```

Reply:
hmm yea i think that is a better pattern, but i'm fine leaving it either way